### PR TITLE
Upgrade EUI, show version on tooltip

### LINKED ISSUE #203: Add version number to main page
Add version to the ems-landing page somewhere, so we know which version of the data we are looking at. Similar to how we show the version in the online docs.

### LINKED ISSUE #603: Upgrade to EUI 80 or newer
I'm not sure if this will need also code updates since it has been a while since we updated our UI framework.

### PR DESCRIPTION
**Pull request by jsanz**
Fixes #603
Fixes #203

* Upgraded to `@elastic/eui@89.1.0` without any further code change
* Added a console message on load to show the EMS version
* Added a tooltip to the title logo to show the EMS version (with a long delay to not annoy casual users)

### CODE REVIEW

File: public/js/components/app.js [hunk 1]
```diff
@@ -321,9 +330,12 @@ export class App extends Component {
       <EuiProvider colorMode="light">
         <EuiHeader>
           <EuiHeaderSectionItem border="right">
-            <EuiHeaderLogo href="/" aria-label={`${this.props.serviceName} home`} iconType="emsApp" >
-              {this.props.serviceName}
-            </EuiHeaderLogo>
+            <EuiToolTip delay="long" 
+              content={`EMS version: ${this.props.client._emsVersion}`}>
```
Comment (nickpeihl) [inline public/js/components/app.js:334]: https://github.com/elastic/ems-client/blob/master/src/ems_client.ts#L216 🙈 

not blocking this, but we should probably add a public method to ems-client to get this private property.

**Review by nickpeihl — APPROVED** [review]
lgtm!

code review and tested with `yarn dev`

File: public/js/components/app.js (hunk 1 above)
Comment (jsanz) [reply public/js/components/app.js:334]: Addressed at https://github.com/elastic/ems-client/pull/204

**Comment by jsanz** [comment]
## 💔 Some backports could not be created

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|v8.12|[](https://github.com/elastic/ems-landing-page/pull/682)|
|✅|v8.11|[](https://github.com/elastic/ems-landing-page/pull/683)|
|✅|v8.10|[](https://github.com/elastic/ems-landing-page/pull/684)|
|❌|v7.17|Conflict resolution was aborted by the user|

### Manual backport
To create the backport manually run:
```
backport --pr 680
```

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Comment by jsanz** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|v7.17|[](https://github.com/elastic/ems-landing-page/pull/685)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**